src/components/table-of-contents.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+
+const TableOfContents = () => (
+  <div class="cmp-table-of-contents">
+    <h2 class="font-chrome font-chrome--small"><span data-text="Table of Contents">Table of Contents</span></h2>
+    <div class="cmp-table-of-contents__items">
+      <ol class="cmp-table-of-contents__list">
+        <div class="cmp-table-of-contents__grid-child">
```

Comment:
`ol`’s are only allowed one kind of child, `li`. You should be able to accomplish this layout with Grid and not need extra `div` elements. If you must, then separate this into two `ol` elements, and on the second `ol` use the `start` attribute to have it semantically pick up at number 4. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol

Reply:
Great catch. The reason I had it this way originally was to get those numbers in the right order. Turns out if I declare `grid-template-rows` in addition to the existing `grid-template-columns`, I can then use the `grid-auto-flow: column` rule to reflow the content!

Reply:
Try CSS columns here instead!